api.c

```diff
@@ -550,6 +561,8 @@ int tcmu_emulate_evpd_inquiry(
 	}
 	break;
 	default:
+		tcmu_err("Vital product data page code 0x%x not support\n",
```

Comment:
Can you use the dev loggers when possible? tcmu_dev_err(dev, ...) here and tcmu_dev_dbg below.

I have a patch that converts everyone, but for new cases it would help if you can use them.